definitions/variable/macro-economy/finance.yaml

```diff
@@ -0,0 +1,8 @@
+- Debt:
+    description: Total net external debt
+    unit: billion USD_2010/yr
+    tier: 2
+- Debt|Public:
```

Comment:
I'm wondering whether it wouldn't make sense to rename to `Debt|Government` to be inline with 'Revenue|Government' and 'Expenditure|Government'. This seems to be a new variable (not yet used in NAVIGATE and ENGAGE), so this shouldn't be much of a hassle for modeling teams?